ci-operator/jobs/operator-framework/operator-lifecycle-manager/operator-framework-operator-lifecycle-manager-master-periodics.yaml

```diff
@@ -32,7 +32,37 @@ periodics:
       - name: JOB_NAME_SAFE
         value: e2e-operator-metrics
       - name: TEST_COMMAND
-        value: make e2e-operator-metrics
+        value: |
+          #!/bin/bash
+
+          git clone https://github.com/openshift/origin.git -b master
+          cd origin
+
+          export GO111MODULE=off
+
+          # install Kubectl required by the test.
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl
+          chmod +x ./kubectl
+          mkdir -p /tmp/shared
+          export PATH=$PATH:/tmp/shared
+          mv ./kubectl /tmp/shared/kubectl
+
+          # make and run test
+          make WHAT=cmd/openshift-tests
+
+          cd ..
+          DIR="./origin/_output/local/bin/$(go env GOHOSTOS)/$(go env GOHOSTARCH)"
+
+          # OLM sanity tests
+          ${DIR}/openshift-tests run all --dry-run | grep "\[sig-operator\].*OLM" | ${DIR}/openshift-tests run --junit-dir=${ARTIFACT_DIR} -f -
+
+          # Install and uninstall rh-operators and run metric tests after
+          EXIT_CODE=0
+          make e2e-operator-metrics || EXIT_CODE=1
+
+          ${DIR}/openshift-tests run all --dry-run | grep "\[sig-instrumentation\].*Prometheus.*should report telemetry if a cloud.openshift.com token is present" | ${DIR}/openshift-tests run --junit-dir=${ARTIFACT_DIR} -f -
```

Comment:
Whats the reason for running this twice?

Reply:
test `\[sig-instrumentation\].*Prometheus.*should report telemetry if a cloud.openshift.com token is present` needs to run first and the rest should run after this test.

Reply:
if you look at the full command, they each specify different tests.

Reply:
I see, okay makes sense, thanks!